### PR TITLE
Clockcult bandaid patch: Anointment oil, abscondence breaches, warp restrictions

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: The chaplain's locker now starts with a flask of anointment oil. It can be splashed onto any container of water to convert it to holy water at half efficiency, as well as normal oil into more anointment oil. This can be done by anyone! The chaplain can also bless containers with oil to turn it into anointment oil.
add: Anointment oil will stifle cult powers; blood rites will be limited, and clockwork cultists can't recite scripture. It metabolizes normally in this state.
add: Cultists and servants with anointment oil in their system will be deconverted 10x faster, taking six ticks instead of sixty.
add: When normal crew have anointment oil in their system, it will metabolize very slowly and give them the ability to see blessed tiles.
add: When the chaplain whacks someone with their bible, they will briefly gain a small amount of anointment oil in their bloodstream if they didn't have it already.
balance: Clockwork cultists can now only teleport into public areas and maintenance. No more warping into the middle of brig.
balance: Abscond now leaves behind an "abscondence breach" for ten seconds, which anyone can interact with to pull the invoker back through after a short delay.
tweak: The sigil of submission now spawns a new slab when it converts someone.
/:cl:

This absolutely doesn't fix every problem clockcult has, but it's a step further towards making it more fun for everyone. I'm still brainstorming ways to improve basebuilding as well as to make it more viable, as right now numbers are more important than fortifications and I'd like to swap that around. Feedback is welcomed and encouraged!

The anointment oil flask can't be directly picked up or dragged by servants and cultists, so they can't directly just snag it and leave with no evidence of sabotage. They have to either drain the flask directly or move the entire locker, which leaves obvious evidence of sabotage.

I also removed the sound spam when the Eminence walks into blessed turfs, and lowered the volume of the Ark's death sound, and made the sigil of submission spawn a slab when it converts someone.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
Mass produce better holy water without ever needing a chaplain except for the initial blessing:thinking:?

**Comment by Xhuis** [comment]
Anointment oil can't bless tiles on its own; it just produces holy water with itself. It also can't deconvert. It's meant to be a useful tool that combines with holy water (another useful tool) to make a powerful weapon against the cult that the crew can use if they're smart about it. If you have any ideas as to how to restrict it, feel free to share.

**Comment by Potato-Masher** [comment]
Ooof. Those abscondence breaches are going to be pretty rough. I guess this is to deter servants from warping away in really obvious spots and/or for toning back the whole "at first sign of danger fuck off back to rebee" stuff.

**Comment by astralloop** [comment]
> add: When the chaplain whacks someone with their bible, they will briefly gain a small amount of anointment oil in their bloodstream if they didn't have it already.
> add: Cultists and servants with anointment oil in their system will be deconverted 10x faster, taking six ticks instead of sixty.

new meta: armored chaplains smacking people guarding the ark with their bible

**Comment by astralloop** [comment]
> balance: Abscond now leaves behind an "abscondence breach" for ten seconds, which anyone can interact with to pull the invoker back through after a short delay.

this is bad. you've remove any ability clockcult has to run and escape from sec

**Comment by Xhuis** [comment]
It's rather difficult to strike balance between "free, immediate escape" and "weak escape that should only be used in hiding." I'm willing to hear all suggestions about it. Shortening it to say, five seconds, would also work, but then it would only have a 3.5 second window for people to yank them back since it takes a 1.5s channel to pull through.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1800,4 +1808,56 @@
...
+	..()
+	if(iscultist(L) || is_servant_of_ratvar(L))
+		to_chat(L, "<span class='notice'>Your distance from your deity returns to normal.</span>")
+	else
+		to_chat(L, "<span class='danger'>Whatever you were closer to, your connection to it fades.</span>")
+		for(var/datum/atom_hud/alternate_appearance/basic/blessedAware/B in GLOB.active_alternate_appearances)
+			B.remove_hud_from(L)
+
+/datum/reagent/anointment_oil/reaction_obj(obj/O, reac_volume) //anointment oil can bless holy water and anoint more of itself, but inefficiently
+	if(O.reagents)
```
Comment (DaedalusGame) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >splashing a container changes the liquid within

this is absolutely brilliant. I'm blown away by this.

**Comment by XDTM** [comment]
If #35639 is merged make sure to make anointment oil give the HOLY trait

**Comment by Plapatin** [comment]
make it to where non-cultists can't walk freely past and behind the barrier at reebe. seen a lot of camping strats done like this and it doesnt feel very fair when dead cultists are dragged past the border to where they're literally unreachable by living ones

**Comment by ExcessiveUseOfCobblestone** [comment]
0.02 metab rate, 0 downsides for something that not only stops cultists from being able to do anything culty but also protects individuals from any cult abilities.

Is that honestly the only way that cult will be enjoyable is if you take cult powers out of the equation entirely? I think it's a bit silly not to think players (at least the ones you want on your cult team) aren't going to be gulping this in bulk as soon as cult is confirmed. Bonus points if they smoke it up with holy water so any individual unfortunately caught in the blast is deconverted nigh-instantly with no way to even inform the rest (because haha nothing works now!) or have enough time to get into a position where they could stop the process.

I don't want to suggest to "make it better" because there's obviously a serious problem with either the abilities or the direction of the mode if this is the most immediate solution you could come up with, and an answer to either of those issues is certainly not this.

**Comment by Xhuis** [comment]
The anointment oil doesn't protect anyone from cult abilities. All it does is give them sight of blessed tiles, which is why it has the very low metabolization rate. In cultists, it has a normal metabolization rate.

**Comment by MoreRobustThanYou** [comment]
Make Bible anointment oil not deconvert

**Comment by Xhuis** [comment]
Anointment oil doesn't deconvert. It just amplifies holy water's decon effect.

**Comment by MoreRobustThanYou** [comment]
Oh

**Comment by MoreRobustThanYou** [comment]
Make it so the anointment oils amplification is reduced when on cult tiles or in the presence of unrestrained cultists, so you still have a chance to rescue them

**Comment by ExcessiveUseOfCobblestone** [comment]
That’s why i should spend a few more seconds reading code and not changelogs!!!

>amps deconversion

By an unholy (heh) amount grr!!!

**Comment by MoreRobustThanYou** [comment]
6 ticks is actually pretty short. Faster than morphine knockout

**Comment by Xhuis** [comment]
Six ticks is extremely short. I think I'll tweak some numbers when I get the chance, mainly by making abscondence breaches last for less time and bumping up the anointment oil conversion time to, say, 10 ticks or so, while halting its effects on cult turfs. Do keep in mind, though, that you can still cleanse holy water from allies by using your ritual knife or using Sentinel's Compromise on them, meaning that a fast-acting team is still likely able to rescue you.

**Comment by DaedalusGame** [comment]
6 ticks is 12 seconds roughly

**Comment by ExcessiveUseOfCobblestone** [comment]
Does clock even have a way to purge holy water outside of purging chems? Bloodcult used to just be hitting yourself with tome but idk if that's still a thing.

**Comment by XDTM** [comment]
They can use their compromise spell

**Comment by Xhuis** [comment]
Sentinel's Compromise cleanses holy water yeah

**Comment by Xhuis** [comment]
Pending some more thought on how to improve stuff, I'll be closing this PR and revising the changes to open in another one. Thanks for y'all's input.

**Comment by AdamElTablawy** [comment]
the lack of downsides and easy massproducing of annointment oil will lead to everything that clock cultists can teleport to being holy watered in short order, since it’s only public areas and maintenence they can’t see into.

annointment oil will also make deconversion mid combat trivially easy. At least if you get syringe gunned with holy water normally you can cure yourself if you don’t go down: this means that ecigs of holy water annointment vape will make you an AoE hazard for any poor cultist in radius, let alone shit like smoke grenades or spraybottles of this stuff.

the breaches will make sneaking around as clock cult even harder, so you get spotted by the crew immediately, and good luck disabling science now that you can’t warp in or rely on marauders to disable it and thus you die to mech hordes or xenobio hordes.

Aside from that annointment oil makes killing the chaplain virtually worthless since sec can still easily deconvert and get holy water even with him gone: now the only thing you lost is his null rod. 

One of these nerfs might have been ok. All of them at once is overkill

**Comment by Xhuis** [comment]
See, one issue here is that I get mixed signals like this constantly. I've been told a lot that Abscond is too powerful because people can warp out on a whim; if I prevent that, it's now useless. If the chaplain goes braindead or dies to an inane way early on, the crew is severely wounded; if I add a workaround, they're now useless. Dancing the balance dance has never been one of my fortes, especially because when I find something fun (and I find clockcult fun) I might not recognize its flaws well. I haven't been not taking action for so long because I don't care, but because I don't know the best solutions for these problems.

**Comment by davethwave** [comment]
Perhaps if this gets passed through you might want to buff up the clockwork side of things like perhaps a new structure that could be useful or allowing the ability to create new clockcult drones in or out of the base.

**Pull request closed without merging**